Submitted By:            Armin K. <krejzi at email dot com>
Date:                    2013-01-24
Initial Package Version: 1.6.1
Upstream Status:         BLFS Specific
Origin:                  Self
Description:             Fix permissions for installed files and disable compression of man pages.

--- a/config-scripts/cups-manpages.m4	2012-04-23 19:26:57.000000000 +0200
+++ b/config-scripts/cups-manpages.m4	2013-01-24 14:17:17.826342931 +0100
@@ -69,10 +69,10 @@
 		;;
 	Linux* | GNU* | Darwin*)
 		# Linux, GNU Hurd, and OS X
-		MAN1EXT=1.gz
-		MAN5EXT=5.gz
-		MAN7EXT=7.gz
-		MAN8EXT=8.gz
+		MAN1EXT=1
+		MAN5EXT=5
+		MAN7EXT=7
+		MAN8EXT=8
 		MAN8DIR=8
 		;;
 	*)
--- a/Makedefs.in	2012-05-23 01:58:31.000000000 +0200
+++ b/Makedefs.in	2013-01-24 14:19:01.380040099 +0100
@@ -40,14 +40,14 @@
 # Installation programs...
 #
 
-INSTALL_BIN	=	$(LIBTOOL) $(INSTALL) -c -m 555 @INSTALL_STRIP@
-INSTALL_COMPDATA =	$(INSTALL) -c -m 444 @INSTALL_GZIP@
+INSTALL_BIN	=	$(LIBTOOL) $(INSTALL) -c -m 755 @INSTALL_STRIP@
+INSTALL_COMPDATA =	$(INSTALL) -c -m 644 @INSTALL_GZIP@
 INSTALL_CONFIG	=	$(INSTALL) -c -m @CUPS_CONFIG_FILE_PERM@
-INSTALL_DATA	=	$(INSTALL) -c -m 444
+INSTALL_DATA	=	$(INSTALL) -c -m 644
 INSTALL_DIR	=	$(INSTALL) -d
-INSTALL_LIB	=	$(LIBTOOL) $(INSTALL) -c -m 555 @INSTALL_STRIP@
-INSTALL_MAN	=	$(INSTALL) -c -m 444
-INSTALL_SCRIPT	=	$(INSTALL) -c -m 555
+INSTALL_LIB	=	$(LIBTOOL) $(INSTALL) -c -m 755 @INSTALL_STRIP@
+INSTALL_MAN	=	$(INSTALL) -c -m 644
+INSTALL_SCRIPT	=	$(INSTALL) -c -m 755
 
 #
 # Default user, group, and system groups for the scheduler...
